lib/resources/Balance.js

```diff
@@ -9,16 +9,4 @@ module.exports = StripeResource.extend({
   retrieve: stripeMethod({
     method: 'GET',
   }),
-
-  listTransactions: stripeMethod({
-    method: 'GET',
-    path: 'history',
-    methodType: 'list',
-  }),
-
-  retrieveTransaction: stripeMethod({
```

Comment:
Wait, I thought we gaveled on keeping things like this?

Reply:
In the general case of nested resources, yes, we're keeping both `stripe.parents.retrieveNested()` and `stripe.nested.retrieve()`. But balance transactions are not a nested resource, they were just historically implemented as one but I think that was an inconsistency we should fix here.

Reply:
Ah, makes sense! Thanks. 

cc @irace-stripe